packages/govern-subgraph/src/GovernQueue.ts

```diff
@@ -95,7 +93,7 @@ export function handleChallenged(event: ChallengedEvent): void {
 export function handleResolved(event: ResolvedEvent): void {
   let container = loadOrCreateContainer(event.params.containerHash)
 
-  container.state = event.params.approved ? EXECUTED_STATUS : CANCELLED_STATUS
+  container.state = event.params.approved ? APPROVED_STATUS : REJECTED_STATUS
```

Comment:
If I'm right should it be ``EXECUTED_STATUS`` and ``CANCELLED_STATUS`` or can you elaborate closer why not?

Reply:
This is where the `resolved` event gets thrown. https://github.com/aragon/govern/blob/2557caba58170c4625e07d19f1f27202dc19d139/packages/govern-core/contracts/pipelines/GovernQueue.sol#L212  

As we can see, the third parameter is `ArbitratorApproved`. So, it's all about approving or rejecting and not executing/cancelling.

Reply:
Thanks for the discussion on Discord. Anythings looks good 👍